Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -304,33 +304,43 @@ public class DateRange
         /// Matches repositories with regards to the <param name="date"/>.
         /// We will use the <param name="op"/> to see what operator will be applied to the date qualifier
         /// </summary>
-        public DateRange(DateTime date, SearchQualifierOperator op)
+        [Obsolete("This ctor has been deprecated as GitHub API now supports date and time")]
+        public DateRange(DateTime date, SearchQualifierOperator op) : this(new DateTimeOffset(date), op)
+        {
+        }
+
+        /// <summary>
+        /// Matches repositories with regards to both the <param name="from"/> and <param name="to"/> dates.
+        /// </summary>
+        [Obsolete("This ctor has been deprecated as GitHub API now supports date and time")]
+        public DateRange(DateTime from, DateTime to) : this(new DateTimeOffset(from), new DateTimeOffset(to))
+        {
+        }
+
+        public DateRange(DateTimeOffset from, DateTimeOffset to)
+        {
+            query = string.Format(CultureInfo.InvariantCulture, "{0:yyyy-MM-dd'T'HH:mm:ss zzz}..{1:yyyy-MM-dd'T'HH:mm:ss zzz}", from.DateTime, to.DateTime);
+        }
+
+        public DateRange(DateTimeOffset date, SearchQualifierOperator op)
         {
             switch (op)
             {
                 case SearchQualifierOperator.GreaterThan:
-                    query = string.Format(CultureInfo.InvariantCulture, ">{0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, ">{0:yyyy-MM-dd'T'HH:mm:ss zzz}", date);
```

Comment:
I'd probably store this pattern in a constant and reuse it throughout.

Reply:
Hi @khellang I've included this suggestion aswell.

I've also updated the relevant unit tests to use a Theory which tests both the `DateTime` and `DateTimeOffset` strategies.